modules/effects/migrations/9_0_0/index.spec.ts

```diff
@@ -0,0 +1,146 @@
+import { Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import * as path from 'path';
+import { createPackageJson } from '../../../schematics-core/testing/create-package';
+
+describe('Effects Migration 9_0_0', () => {
+  let appTree: UnitTestTree;
+  const collectionPath = path.join(__dirname, '../migration.json');
+  const pkgName = 'effects';
+
+  beforeEach(() => {
+    appTree = new UnitTestTree(Tree.empty());
+    appTree.create(
+      '/tsconfig.json',
+      `
+        {
+          "include": [**./*.ts"]
+        }
+       `
+    );
+    createPackageJson('', pkgName, appTree);
+  });
+
+  describe('Replaces resubscribeOnError with useEffectsErrorHandler in effect options', () => {
+    describe('should replace resubscribeOnError configuration key with useEffectsErrorHandler', () => {
```

Comment:
this describe can be left out? It has more or less the same description as the parent

Reply:
ah actually you've picked up a mistake in my describe nesting. the structure should be

```
Effects Migration 9_0_0
└── Replaces resubscribeOnError with useEffectsErrorHandler in effect options
    ├── should replace resubscribeOnError configuration key with useEffectsErrorHandler
    |   ├── in createEffect() effect creator
    |   ├── in @Effect() decorator
    └── should not replace non-ngrx identifiers
        ├── in module scope
        └── within create effect callback
```

fixed.